events/2018/12-contributor-summit/README.md

```diff
@@ -0,0 +1,45 @@
+# 2018 Kubernetes Contributor Summit US
+
+## What
+
+The Contributor Summit brings together new and current Kubernetes contributors alike to connect and share face-to-face. It is an opportunity for existing contributors to help shape the future of community development and offers a welcoming space for new community members to learn, explore and put the contributor workflow to practice.
+The summit now spans two days; including an optional social event in the early evening of December 9th along with the main full-day event December 10th.
+
+In some sense, the summit is a real-life extension of the community meetings and SIG meetings. There are three explicit goals:
+
+- Cross-SIG communication and announcements
+- Grow the number of new developers
+- Send contributors home with more context, knowledge, and skills about the project
+
+## Registration
+
+- Register for KubeCon and select the Contributor Summit from the list of co-located events
+- If you are planning on attending the New Contributor Track, Sign the CLA if you have not done so already. 
```

Comment:
You may want to link to registration itself and the CLA DOC
* [kubecon registration](https://www.regonline.com/registration/Checkin.aspx?EventID=2246960)
* [CLA Doc](https://github.com/kubernetes/community/blob/master/CLA.md)